src/Razor/Microsoft.CodeAnalysis.Razor/src/TextChangeExtensions.cs

```diff
@@ -11,11 +11,6 @@ internal static class TextChangeExtensions
     {
         public static SourceChange AsSourceChange(this TextChange textChange)
         {
-            if (textChange == null)
-            {
-                throw new ArgumentNullException(nameof(textChange));
-            }
-
```

Comment:
Huh?

Reply:
Nullability errors. Since textChange isn't nullable, this condition is superfluous.